lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -209,6 +220,24 @@ def self.available_options
                                 UI.user_error!("No App name given, pass using `app_name: 'app name'`") unless value && !value.empty?
                               end),
 
+          FastlaneCore::ConfigItem.new(key: :app_display_name,
+                                  env_name: "APPCENTER_APP_DISPLAY_NAME",
+                               description: "App display name. Used for new app creation, if app with 'app_name' name was not found",
```

Comment:
🙃 Potentially "App display name to use when creating a new app" is simpler.
The logic that it creates a new app if app with `app_name` is not found might also be added to that config items description.

Reply:
👍 I agree, thanks!